### ISSUE TITLE
Simple Share prevents the “Widgets” section from appearing in the Customizer

### ISSUE THREAD

**Issue by nickcernis** [copyblogger/genesis-simple-share#49]
Under certain circumstances, Genesis Simple Share can prevent the Widgets area from appearing in the Customizer.
#### Steps to reproduce

As discovered by @dreamwhisper:
1. Activate Genesis Simple Share and the Altitude Pro theme (or any theme with a `front-page.php` file).
2. Check that the Genesis → Simple Share settings are set to show on posts:
   
   ![genesis_simple_share_settings_ _my_blog_ _wordpress](https://cloud.githubusercontent.com/assets/647669/7416656/e1151156-ef58-11e4-9b09-614cdfb60a37.jpg)
3. Set the homepage to use a Static Page in the Customizer:
   
   ![customize__loading](https://cloud.githubusercontent.com/assets/647669/7416540/13a8ced8-ef58-11e4-8a05-fbb71328f0dd.jpg)
4. Add a Genesis Featured Posts widget to the Front Page 1 area:
   
   ![customize__about_-_my_blog](https://cloud.githubusercontent.com/assets/647669/7417147/f9ef4d74-ef5b-11e4-9d6c-b097cb0cdeae.jpg)
5. Save the changes in the Customizer and refresh the Customizer page.
#### Actual behaviour

The Widgets area will be missing from the Customizer (WordPress 4.2.1) or empty (Rainmaker), and the console displays this error:

> Uncaught TypeError: $(...).waypoint is not a function

**Missing Widgets area in WP**:

![customize__loading](https://cloud.githubusercontent.com/assets/647669/7417019/2fc78dae-ef5b-11e4-8ff3-58a9bbc18ecc.jpg)

**Blank Content Areas section in Rainmaker**: (same JS error in console)

![customize_altitude_pro_theme_ _rainmaker](https://cloud.githubusercontent.com/assets/647669/7417101/a291d538-ef5b-11e4-959b-c2b58b76c1d1.jpg)

If you change “front page displays” to “Your Latest Posts”, uncheck “post” in the Simple Share Settings, or remove the Featured Post widget, the Widgets area becomes visible in the Customizer again:

![customize__about_-_my_blog](https://cloud.githubusercontent.com/assets/647669/7417043/5bfb86aa-ef5b-11e4-8a04-d1b2d816f6ae.jpg)
#### Expected behaviour

The Widgets area should be visible in the Customizer regardless of the widgets used or Static Front Page and Simple Share settings.
#### Suggested fix

Enqueueing the waypoints JS on [line 138 of front-end.php](https://github.com/copyblogger/genesis-simple-share/blob/master/lib/front-end.php#L138) if the Customizer is active resolves this in my tests. (I tested WP 4.2.1; I haven't yet tested Rainmaker.)

**Before:**

``` php
if( $this->is_archive() ) {
    wp_enqueue_script( 'genesis-simple-share-waypoint-js' );
}
```

**After:**

``` php
if( $this->is_archive() || is_customize_preview() ) {
    wp_enqueue_script( 'genesis-simple-share-waypoint-js' );
}
```

**Comment by carasmo** [comment]
Just wondering if this will get fixed soon.

**Comment by NicktheGeek** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Genesis Simple Share](https://trello.com/c/p4ZJkg29/10-genesis-simple-share)

**Comment by carasmo** [comment]
any news on this?

**Comment by NicktheGeek** [comment]
@carasmo yes! This should be fixed in the 1.1 release. It is being tested, documented, and getting new design assets for the wp.org repo right now but we have a release candidate ready if you'd like to test it.

https://github.com/copyblogger/genesis-simple-share/releases/tag/1.1rc2

**Comment by dreamwhisper** [comment]
Confirmed x3
